### DISCUSSION TITLE
Playwright scroll node into view

### DISCUSSION THREAD

**Discussion by bottbott** [xyflow/xyflow#4124] [Q&A]
Hi there, 

I was wondering if someone could give me a pointer on how to scroll a node into view that is outside of the viewable area. I'm using Playwright for testing and the UI mode is showing the click event trying to occur on my first node but its relative position has it hidden underneath the sidebar. You can see the red circle where its trying to click. 

![image](https://github.com/xyflow/xyflow/assets/53452222/473edac4-7163-4499-b342-844a309d6700)

I'm just making a call to 
`await page.click('.react-flow__node');`
But it doesn't seem to be enough to scroll in to view as well. 

Appreciate any ideas! Cheers!

**Comment by moklick** [comment]
Does `.click({ force: true })` work for you?
